src/DotNet/Writer/IWriterError.cs

```diff
@@ -11,6 +11,7 @@ public interface IWriterError {
 		/// can be ignored but the written data won't be valid.
 		/// </summary>
 		/// <param name="message">Error message</param>
-		void Error(string message);
+		/// <param name="args">Optional message arguments</param>
+		void Error(string message, params object[] args);
```

Comment:
This is a breaking change.

Reply:
It's compatible on source level, do you mean breaking change on binary?